config.json

```diff
@@ -349,7 +349,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Concise",
+      "content": "Raku programs are naturally more concise and readable, thanks to its innovative design.",
+      "icon": "powerful"
+    },
+    {
+      "title": "Expressive",
+      "content": "Raku is a supremely flexible language, adaptable to your own style of programming.",
+      "icon": "fun"
+    },
+    {
+      "title": "Growing",
+      "content": "Raku is developed by a team of dedicated and enthusiastic open source volunteers.",
+      "icon": "evolving"
+    },
+    {
+      "title": "Concurrency",
+      "content": "Raku has a high level concurrency model, both for implicit as well as explicit multiprocessing.",
+      "icon": "concurrency"
+    },
+    {
+      "title": "Helpful Documentation",
```

Comment:
I am not sure about the content, alas, but documentation is something that hopefully exists for every language out there, so maybe another feature or trait would be nicer to show? Or it's just a way to give a link?

Reply:
Is there any feature in particular you'd like to highlight? Grammars might be a good one off the top of my head.